python/pyarrow/tensor.pxi

```diff
@@ -199,7 +202,13 @@ shape: {0.shape}""".format(self)
             for x in dim_names:
                 c_dim_names.push_back(tobytes(x))
 
-        coords = np.vstack([obj.row, obj.col]).T
+        row = obj.row
+        col = obj.col
+        if obj.has_canonical_format:
```

Comment:
Does this attribute exist? I don't see it in https://docs.scipy.org/doc/scipy/reference/generated/scipy.sparse.coo_matrix.html#scipy.sparse.coo_matrix
In https://docs.scipy.org/doc/scipy/reference/generated/scipy.sparse.csr_matrix.html#scipy.sparse.csr_matrix, I see an attribute named `has_sorted_indices`

Reply:
The attribute [actually exists](https://github.com/scipy/scipy/blob/master/scipy/sparse/coo.py#L137) but not documented.  It may be an internal attribute, but it does not have `_` prefix, so I detect it can be used.  I need to refer this attribute to recognize whether or not a given SciPy's coo_matrix has canonical format.